tests/integration/galley/analyze_test.go

```diff
@@ -76,15 +77,10 @@ func TestFileOnly(t *testing.T) {
 
 			istioCtl := istioctl.NewOrFail(ctx, ctx, istioctl.Config{})
 
-			// Validation error if we have a service role binding without a service role
-			output, err := istioctlSafe(t, istioCtl, ns.Name(), false, serviceRoleBindingFile)
+			// Validation error if we have a gateway with invalid selector
+			output, err := istioctlSafe(t, istioCtl, ns.Name(), false, gatewayFile)
 			expectMessages(t, g, output, msg.ReferencedResourceNotFound)
 			g.Expect(err).To(BeIdenticalTo(analyzerFoundIssuesError))
-
-			// Error goes away if we include both the binding and its role
-			output, err = istioctlSafe(t, istioCtl, ns.Name(), false, serviceRoleBindingFile, serviceRoleFile)
```

Comment:
this is changing the test, we no longer verify we can fix errors

Reply:
thanks, fixed.